Source/Atomic/Network/Network.cpp

```diff
@@ -203,6 +204,42 @@ void Network::ClientDisconnected(kNet::MessageConnection* connection)
     }
 }
 
+bool Network::ConnectSimple(const String& address, unsigned short port, Scene* scene)
+{

```

Comment:
Once we have Variant maps exposed on the methods (to javascript), we won't need this.